src/govuk/components/modal-dialogue/template.njk

```diff
@@ -0,0 +1,69 @@
+          ></path>
+          {#- Fallback PNG image for older browsers.
+          The <image> element is a valid SVG element. In SVG, you would specify
+          the URL of the image file with the xlink:href – as we don't reference an
+          image it has no effect. It's important to include the empty xlink:href
+          attribute as this prevents versions of IE which support SVG from
+          downloading the fallback image when they don't need to.
+          In other browsers <image> is synonymous for the <img> tag and will be
+          interpreted as such, displaying the fallback image. #}
+          <image src="{{ params.assetsPath | default('/assets/images') }}/govuk-logotype-crown.png" xlink:href="" class="govuk-header__logotype-crown-fallback-image" width="36" height="32"></image>
+        </svg>
+
+        {{ govukButton({
+          text: "×",
+          type: "button",
+          classes: "govuk-modal-dialogue__close",
+          attributes: {
+            "aria-label": "Close modal dialogue"
```

Comment:
I wonder if all users will know what is meant by "modal dialogue"? The users of GOV.UK Publishing (which this is based on) would mainly be repeat users with at least a moderate level of computer literacy and are therefore not representative of the wider population.

If we could test with an alternative wording that would be great.